executor/aggfuncs/aggfunc_test.go

```diff
@@ -592,8 +697,7 @@ func (s *testSuite) testAggMemFunc(c *C, p aggMemTest) {
 	iter := chunk.NewIterator4Chunk(srcChk)
 	i := 0
 	for row := iter.Begin(); row != iter.End(); row = iter.Next() {
-		memDelta, err := finalFunc.UpdatePartialResult(s.ctx, []chunk.Row{row}, finalPr)
-		c.Assert(err, IsNil)
```

Comment:
Should we keep this error check?

Reply:
I don't know, this isn't my code, I'm trying rebase.

Reply:
ok

Reply:
The conflicting file `executor/aggfuncs/aggfunc_test.go` is resolved. but there are some wrong in `executor.go`.